File: itime.go
package main

import (
    "fmt"
    "time"
)

func main(){
    thetime := time.Now().UTC()
    thehours := float32(thetime.Hour())
    theminutes := float32(thetime.Minute())
    theseconds := float32(thetime.Second())

if thehours == 23 {
	thehours = 0
} else {
	thehours = thehours + 1
}

thebeats := ((((thehours*60)+theminutes)*60) + theseconds) / 86.4
fmt.Printf("@%.2f", thebeats)
}

File: itime.nim
import times, strutils

let utctime = now().utc
var thehours: int

if ((utctime.hour) mod 24 == 23):
    thehours = 0
else:
    thehours = ((utctime.hour) mod 24)+1

let theminutes = utctime.minute
let theseconds = utctime.second
var thebeats = ((((float(thehours)*60)+float(theminutes))*60) + float(theseconds)) / 86.4


echo "@" & thebeats.formatFloat(ffDecimal, 2)
